tx-builder/marginfi/buildSell.ts

```
@@ -0,0 +1,369 @@
+                if (!targetBalance) {
+                    throw Error(
+                        `Balance for bank ${targetBank.toBase58()} should be projected active at this point (ix ${index}: ${decoded.name
+                        }))`
+                    );
+                }
+
+                if (ixArgs.repayAll || ixArgs.withdrawAll) {
+                    targetBalance.active = false;
+                    targetBalance.bankPk = PublicKey.default;
+                }
+            }
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)